semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -351,6 +351,42 @@ and assign env lhs tok rhs_exp e_gen =
       mk_e
         (Composite (composite_of_container ckind, (tok1, tup_elems, tok2)))
         (related_exp lhs)
+  | G.Record (tok1, fields, tok2) ->
+      (* {x1: v1, ..., xN: vN} = RHS *)
+      let tmp = fresh_var env tok2 in
+      let tmp_lval = lval_of_base (Var tmp) in
+      add_instr env (mk_i (Assign (tmp_lval, rhs_exp)) eorig);
```

Comment:
maybe put in a comment how the record you put above {x1: v1, ...} = rhs is transformed in the IL,
including this __FIXME_AST_to_IL__